docs/docsite/rst/user_guide/vault.rst

```diff
@@ -37,8 +54,17 @@ Vault IDs and Multiple Vault Passwords
 *Available since Ansible 2.4*
```

Comment:
In line with the comment from @gundalow about the docs being versioned, I suggest that we take this out.

```suggestion
```